spec/libraries/helpers_spec.rb

```diff
@@ -124,7 +125,7 @@
   context 'centos 5 64 bit' do
```

Comment:
you'll want to update this context description to match the version being passed now

Reply:
Yeah this whole section needs fixing. It's a bit of a jumble